documentation/enhanced_integrations.md

```
@@ -95,6 +95,11 @@ Spook enhances the following {term}`Home Assistant` {term}`integrations <integra
 [![](https://brands.home-assistant.io/select/icon.png)](integrations/select)
 :::
 
+:::{card} Sensor
+:footer: 📚 [Learn more](integrations/sensor)
+[![](https://brands.home-assistant.io/sensor/icon.png)](integrations/sensor)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)